frontend/src/main/pages/LoadSubjectsPage.js

```diff
@@ -26,14 +26,21 @@ const LoadSubjectsPage = () => {
     method: 'POST',
   });
 
+  subjectsCount = subjects.length;
+
   const onSuccess = (subjects) => {
-    if (subjectsCount === subjects.length) {
+    if (subjects.length - subjectsCount === 0) {
       toast('No new subjects were loaded');
     } else {
-      toast(`${subjects.length} subjects loaded`);
-      subjectsCount = subjects.length;
+      if (subjects.length - subjectsCount === 1) {
```

Comment:
idk if this will cause errors or not show properly if "subjects.length - subjectsCount" is more than 1

Reply:
This looks correct - this `if` statement is to handle the case where `subject` has to be singular in the toast message per grammar rules.